pkg/utilities/version/version_test.go

```diff
@@ -1,61 +1,31 @@
-package version_test
+package version
 
 import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"github.com/weaveworks/wksctl/pkg/kubernetes"
-	"github.com/weaveworks/wksctl/pkg/utilities/version"
 )
 
-func TestMatchesRangeDefaultVersion(t *testing.T) {
-	matches, err := version.MatchesRange(kubernetes.DefaultVersion, kubernetes.DefaultVersionsRange)
-	assert.NoError(t, err)
-	assert.True(t, matches)
-}
 func TestMatchesRangeAnyVersionsRangeWildcard(t *testing.T) {
-	matches, err := version.MatchesRange("1.anything", "*")
+	matches, err := MatchesRange("1.anything", "*")
 	assert.NoError(t, err)
 	assert.True(t, matches)
 }
+
 func TestMatchesRangeAnyVersionsRangeBlank(t *testing.T) {
-	matches, err := version.MatchesRange("1.anything", "")
+	matches, err := MatchesRange("1.anything", "")
 	assert.NoError(t, err)
 	assert.True(t, matches)
 }
 
 func TestMatchesInvalidVersionsRange(t *testing.T) {
-	matches, err := version.MatchesRange("1.2.3", "bad range")
+	matches, err := MatchesRange("1.2.3", "bad range")
 	assert.EqualError(t, err, "invalid versions range \"bad range\": Could not get version from string: \"bad\"")
 	assert.False(t, matches)
 }
 
 func TestMatchesInvalidVersion(t *testing.T) {
-	matches, err := version.MatchesRange("bad version", ">1.0.0 <=1.2.3")
+	matches, err := MatchesRange("bad version", ">1.0.0 <=1.2.3")
 	assert.EqualError(t, err, "invalid version \"bad version\": Invalid character(s) found in major number \"bad version\"")
 	assert.False(t, matches)
 }
-
-func TestVersionLessthanWithBothVs(t *testing.T) {
```

Comment:
why did we lose these?

Reply:
`version.LessThan` (and some other `version` functions as well) are used by the machine controller, and have thus been moved to `capi-existinginfra` along with these tests. See [capi-existinginfra/pkg/utilities/version](https://github.com/twelho/capi-existinginfra/tree/master/pkg/utilities/version).